OPERATING.md

```diff
@@ -266,7 +266,7 @@ The adaptor's database records:
 * metadata about the transfer process
 
 The supplier MUST configure the `GP2GP_MONGO_TTL` variable to remove the database records
-after a reasonable time period.
+after a reasonable time period. The specs say 12 weeks, so this is our suggestion.
```

Comment:
I'd propose to change the description to 84 days (e.g. "The specs say 84 days...") to be consistent with the TTL format (P84D). This way it will be less confusing for the users of the adaptor.

Reply:
Same.